docs/source/serving/deploying_with_triton.rst

```diff
@@ -0,0 +1,6 @@
+.. _deploying_with_triton:
+
+Deploying with Triton
```

Comment:
nit:
```suggestion
Deploying with NVIDIA Triton
```

I found some people confuse NVIDIA Triton with OpenAI Triton. I think clarifying this would help. WDYT?